collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/IntentGroupTest.java

```diff
@@ -128,7 +128,7 @@ public void externalApp_ShouldPopulateFields() throws IOException {
         resultIntent.setClipData(clipData);
         resultIntent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
         intending(not(isInternal())).respondWith(new Instrumentation.ActivityResult(Activity.RESULT_OK, resultIntent));
-        onView(withText("This is buttonText")).perform(click());
+        onView(withText("This is buttonText")).perform(nestedScrollTo(), click());
```

Comment:
These tests were failing due to the audio widget getting slightly taller and it forcing other items in the test to be scrolled off screen.